deltachat-ios/DC/Wrapper.swift

```diff
@@ -919,37 +1008,32 @@ class DcMsg: MessageType {
         return dc_msg_get_showpadlock(messagePointer) == 1
     }
 
-    func createChat() -> DcChat {
-        let chatId = dc_create_chat_by_msg_id(mailboxPointer, UInt32(id))
-        return DcChat(id: Int(chatId))
-    }
-
     func sendInChat(id: Int) {
-        dc_send_msg(mailboxPointer, UInt32(id), messagePointer)
+        dc_send_msg(DcContext.shared.contextPointer, UInt32(id), messagePointer)
```

Comment:
we should pass `dcContext` as a parameter instead of using global `DcContext.shared`. this is also true for the other usages of `DcContext.shared.contextPointer` in this file.